packages/fixtures/src/events.spec.ts

```diff
@@ -41,4 +41,36 @@ describe('Events fixtures', () => {
       expect.objectContaining(options),
     );
   });
+
+  it('generates a memberJoinedChannel block', () => {
+    expect.assertions(1);
+
+    expect(events.memberJoinedChannel().event).toEqual(
+      expect.objectContaining({ type: 'member_joined_channel' }),
+    );
```

Comment:
I know we aren't doing this elsewhere in these tests, and fine with this being a separate change, but I think this might be better as a snapshot test. Ensuring the output matches the spec/API example seems like the best option here, not just checking the `type` on `.event`. Also illustrates the output better, especially for those not using TypeScript. 

I do get that the typing helps ensure ^ though.